src/ParallelHashMap.h

```diff
@@ -208,8 +204,8 @@ bool FixedHashMap<K,V>::contains(K key)
  * @return value associated with the given key
  */
 template <class K, class V>
-V& FixedHashMap<K,V>::at(K key)
-{
+V& FixedHashMap<K,V>::at(K key) {
+
   /* get hash into table assuming M is a power of 2, using fast modulus */
   size_t key_hash = std::hash<K>()(key) & (_M-1);
 
```

Comment:
Per the google c++ style guide, can you put braces around the while loop starting on line 214?